src/rpc/handlers/GetAggregatePrice.hpp

```diff
@@ -59,8 +59,8 @@ class GetAggregatePriceHandler {
      * @brief A struct to hold the statistics
      */
     struct Stats {
-        ripple::STAmount avg;
-        ripple::Number sd;  // standard deviation
+        ripple::STAmount avg{};  // NOLINT(readability-redundant-member-init)
+        ripple::Number sd{};     // NOLINT(readability-redundant-member-init). standard deviation
```

Comment:
maybe move `// standard deviation` on it's own line above this field declaration?